assets/src/domain/shared/entities/datetimes/predicates/filters/salesFilter.ts

```diff
@@ -2,26 +2,21 @@ import { Datetime } from '@edtrServices/apollo';
 import { DatetimeSales } from '@edtrServices/filterState';
 
 import aboveCapacity from './aboveCapacity';
-import allDates from './allDates';
 import belowCapacity from './belowCapacity';
-import notTrashed from '../../../../services/predicates/filters/notTrashed';
 
 import { SalesFilter } from './types';
 
 /**
  * reduces dates array based on value of the "sales" filter
  */
-const salesFilter = ({ dates: entities, sales = DatetimeSales.all }: SalesFilter): Datetime[] => {
-	const dates = notTrashed(entities);
```

Comment:
Sales filter should not be concerned about trashed. That is the job of status filter.

Reply:
ya this was initially modeled after what I did in EDTR v1 where, it wasn't as easy to manipulate lists of entities